cloud/fallback-public-cloud.yaml

```diff
@@ -337,14 +337,116 @@ clouds:
+      eu-dcc-dublin-1:
+        endpoint: https://iaas.eu-dcc-dublin-1.oraclecloud.com
+      eu-dcc-dublin-2:
+        endpoint: https://iaas.eu-dcc-dublin-2.oraclecloud.com
+      eu-dcc-milan-1:
+        endpoint: https://iaas.eu-dcc-milan-1.oraclecloud.com
+      eu-dcc-milan-2:
+        endpoint: https://iaas.eu-dcc-milan-2.oraclecloud.com
+      eu-dcc-rating-1:
+        endpoint: https://iaas.eu-dcc-rating-1.oraclecloud.com
+      eu-dcc-rating-2:
+        endpoint: https://iaas.eu-dcc-rating-2.oraclecloud.com
       eu-frankfurt-1:
         endpoint: https://iaas.eu-frankfurt-1.oraclecloud.com
+      eu-frankfurt-2:
+        endpoint: https://iaas.eu-frankfurt-2.oraclecloud.com
+      eu-jovanovac-1:
+        endpoint: https://iaas.eu-jovanovac-1.oraclecloud.com
+      eu-madrid-1:
+        endpoint: https://iaas.eu-madrid-1.oraclecloud.com
+      eu-madrid-2:
+        endpoint: https://iaas.eu-madrid-2.oraclecloud.com
+      eu-marseille-1:
+        endpoint: https://iaas.eu-marseille-1.oraclecloud.com
+      eu-milan-1:
+        endpoint: https://iaas.eu-milan-1.oraclecloud.com
+      eu-paris-1:
+        endpoint: https://iaas.eu-paris-1.oraclecloud.com
+      eu-stockholm-1:
+        endpoint: https://iaas.eu-stockholm-1.oraclecloud.com
+      eu-zurich-1:
+        endpoint: https://iaas.eu-zurich-1.oraclecloud.com
+      il-jerusalem-1:
+        endpoint: https://iaas.il-jerusalem-1.oraclecloud.com
+      me-abudhabi-1:
+        endpoint: https://iaas.me-abudhabi-1.oraclecloud.com
+      me-dcc-muscat-1:
+        endpoint: https://iaas.me-dcc-muscat-1.oraclecloud.com
+      me-dubai-1:
+        endpoint: https://iaas.me-dubai-1.oraclecloud.com
+      me-jeddah-1:
+        endpoint: https://iaas.me-jeddah-1.oraclecloud.com
+      mx-monterrey-1:
+        endpoint: https://iaas.mx-monterrey-1.oraclecloud.com
+      mx-queretaro-1:
+        endpoint: https://iaas.mx-queretaro-1.oraclecloud.com
+      sa-santiago-1:
+        endpoint: https://iaas.sa-santiago-1.oraclecloud.com
+      sa-saopaulo-1:
+        endpoint: https://iaas.sa-saopaulo-1.oraclecloud.com
+      sa-vinhedo-1:
+        endpoint: https://iaas.sa-vinhedo-1.oraclecloud.com
+      uk-cardiff-1:
+        endpoint: https://iaas.uk-cardiff-1.oraclecloud.com
+      uk-gov-cardiff-1:
+        endpoint: https://iaas.uk-gov-cardiff-1.oraclecloud.com
+      uk-gov-london-1:
+        endpoint: https://iaas.uk-gov-london-1.oraclecloud.com
       uk-london-1:
         endpoint: https://iaas.uk-london-1.oraclecloud.com
+      us-ashburn-1:
+        endpoint: https://iaas.us-ashburn-1.oraclecloud.com
+      us-chicago-1:
+        endpoint: https://iaas.us-chicago-1.oraclecloud.com
+      us-gov-ashburn-1:
+        endpoint: https://iaas.us-gov-ashburn-1.oraclecloud.com
+      us-gov-chicago-1:
+        endpoint: https://iaas.us-gov-chicago-1.oraclecloud.com
+      us-gov-phoenix-1:
+        endpoint: https://iaas.us-gov-phoenix-1.oraclecloud.com
```

Comment:
We should investigate these gov regions before adding. We used to have "aws-gov" and "aws-china" as separate clouds, but I only see that in fallback-public-cloud.go file now, not the output of `juju clouds`. I'm not sure how this all works now.